courses/free-courses-hi.md

```diff
@@ -357,6 +357,7 @@
 
 ### iOS
 
+*[iOS App Devlopment for Beginner in Hindi](https://www.youtube.com/playlist?list=PL5PR3UyfTWvcRhz4ms-nNoGja2-DkA_6T) - iOS Academy
```

Comment:
```suggestion
* [Use the title from the Playlist](https://www.youtube.com/playlist?list=PL5PR3UyfTWvcRhz4ms-nNoGja2-DkA_6T) - iOS Academy
```
Don't invent titles! Use the title given by the resource.

Reply:
fixed